### PR TITLE
documentation(EJ2-62685): Added core video.

### PR DESCRIPTION
**Pull request by sharmi70**

### CODE REVIEW

**Review by VigneshNatarajan27 — APPROVED** [review]
changes are fine

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/getting-started-core.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Getting Started with ##Platform_Name## Grid Control
+title: Getting Started with Syncfusion ASP.NET Core Grid Control
```
Comment (MaithiliyK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/getting-started-core.md:3]: this changes is need since it is common for both mvc and core right

**Review by MaithiliyK — COMMENTED** [review]
address my comments

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/getting-started-core.md (hunk 1 above)
Comment (sharmi70) [reply ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/getting-started-core.md:3]: this changes only for core itself. we included changes in this md file, so below CI error is shown.

image

That's why i edited the **title** to resolve the CI issue.

**Review by MaithiliyK — APPROVED** [review]
changes are fine

**Pull request merged**